front/lib/resources/file_resource.ts

```diff
@@ -222,7 +225,14 @@ export class FileResource extends BaseResource<FileModel> {
     return `files/w/${workspaceId}/${fileId}/${version}`;
   }
 
-  async getSignedUrlForDownload(
+  getPublicUrlForDownload(auth: Authenticator): string {
+    return (
+      "https://storage.googleapis.com/" +
```

Comment:
IIRC, the google SDK exposes this already.

Reply:
You can get the right url without building it by doing: `getPublicUploadBucket().file(this.getCloudStoragePath(auth, "public")).publicUrl()`

Reply:
nice! :+1: